Documentation/CRDs/Cluster/external-cluster/topology-for-external-mode.md

```diff
@@ -113,6 +113,13 @@ reclaimPolicy: Delete
 volumeBindingMode: WaitForFirstConsumer
 ```
 
+Update the rook-ceph-operator-config configmap data with the key value,
```

Comment:
The line numbers of those links below will be difficult to maintain. In fact, since last week they are already incorrect. Just put a link to the operator.yaml and they can see the configmap there at the top of the file.
```suggestion
Set two values in the [rook-ceph-operator-config configmap](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml):
```

Reply:
okay make sense, not sure if there is dynamic linking available now or in future...